salt/modules/dockermod.py

```diff
@@ -6728,9 +6727,32 @@ def call(name, function, *args, **kwargs):
         name, thin_path, os.path.join(thin_dest_path, os.path.basename(thin_path))
     )
 
+    # figure out available python interpreter inside the container
+    pycmds = (
+        "python3",
+        "/usr/libexec/platform-python",
+        "python27",
```

Comment:
We no longer support python2. Is there a reason you are including these unsupported versions?

Reply:
I was not sure really what to do here, so I followed the same approach that we currently have in our SSH SHIM at https://github.com/saltstack/salt/blob/master/salt/client/ssh/__init__.py#L148

So, in this case, this would allow `salt-call` to run with any Python version in the container and will let "salt-call" to report the error message about Python 2 is not longer supported since Sodium. (Same approach that for "salt-ssh" atm)

Reply:
I do know that some of the python2 code was kept in salt-ssh to ensure this feature `ssh_ext_alternatives` would still work with older versions of Salt. Since that feature is not something that is supported with docker can we remove the python2 versions?

Reply:
Done :+1: